server/events/command_requirement_handler.go

```diff
@@ -65,6 +67,20 @@ func (a *DefaultCommandRequirementHandler) ValidateApplyProject(repoDir string,
 	return "", nil
 }
 
+func (a *DefaultCommandRequirementHandler) ValidateProjectDependencies(ctx command.ProjectContext) (failure string, err error) {
```

Comment:
`Dependencies` makes sense here for the function name. I know I mentioned renaming some of the variables to `DependsOn`. I'm not going to nit pick on every single one, just use good judgment on which ones make sense to change.